lib/api/open.js

```diff
@@ -0,0 +1,5 @@
+var request = require('../tools/smb2-forge').request;
+
+module.exports = function open(path, cb) {
```

Comment:
openFile ?...

Reply:
This method is the match of `fs.open` (like `read`) but I couldn't use the name `close` due to it being taken by the method closing the connection.

I'm not against changing the name but then we might wonder abouth `read` as well…

Reply:
And there is a readFile somewhere already, if I remember well, hope this won't collide on the same proto...